src/main/php/Sniffs/PHPCodeSniffer/Standards/ZooRoyal/Sniffs/Commenting/DocCommentAlignmentSniff.php

```diff
@@ -162,6 +170,7 @@ public function process(File $phpcsFile, $stackPtr)
                 && $tokens[($i + 1)]['content'] !== ' '
                 && strtolower(substr($tokens[($i + 1)]['content'], 0, 3)) !== '@oa'
                 && strtolower(substr($tokens[($i + 2)]['content'], 0, 3)) !== '@oa'
+                && !str_starts_with(strtolower($tokens[($i + 1)]['content']), '@orm')
```

Comment:
Why using brackets in all the index statements? `$i + 1` should work as expected

Reply:
As this was just a core hack to learn if it helps solving your problem I wouldn't go too deep into that. ^